engine/src/execution_graph/logic_controllers/CacheMachine.h

```diff
@@ -450,6 +457,62 @@ class CacheDataLocalFile : public CacheData {
 	size_t size_in_bytes; /**< The size of the file being stored. */
 };
 
+
+/**
+* A CacheData that stores is data in an ORC file.
```

Comment:
this is incorrectly copy pased commentary. please put correct commentary for what this is